MapboxNavigation/InstructionsCardStyle.swift

```diff
@@ -0,0 +1,39 @@
+@objc public protocol InstructionsCardStyle: class {
```

Comment:
We could break out the core functionality in `Style` to a Stylable protocol. That way, we could extend this card styling in `InstructionsCardStyle` and provide day style and night style for cards.

Reply:
@frederoni - captured in this ticket #2175. Will be resolved in an upcoming PR.